x-pack/legacy/plugins/code/common/repo_file_status.ts

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+export enum RepoFileStatus {
+  LANG_SERVER_IS_INITIALIZING = 'Language server is initializing.',
+  INDEXING = 'Indexing in progress',
+  FILE_NOT_SUPPORTED = 'Current file is not of a supported language.',
```

Comment:
nit: maybe reword to `The programming language is not supported for language servers in this file.`.